README.md

```diff
@@ -17,7 +17,7 @@ This project is inspired by [BitMex Research: Bitcoin Miner Transaction Fee Gath
 
 The miningpool-observer project is built with self-hosting in mind.
 Both private and public instances, like e.g. [miningpool.observer](https://miningpool.observer), are supported.
-Requirements are a Bitcoin Core node v22.0 (currently, you'll need a self-compiled `master` build! requires [PR #18772 (rpc: calculate fees in getblock using BlockUndo data)](https://github.com/bitcoin/bitcoin/pull/18772))) and a PostgreSQL database.
+Requirements are a Bitcoin Core node v23.0 and a PostgreSQL database.
```

Comment:
I think I wrote this documentation before v22.0 was released so v22.0 works too. The v22.0 [release notes](https://raw.githubusercontent.com/bitcoin/bitcoin/master/doc/release-notes/release-notes-22.0.md ) contain https://github.com/bitcoin/bitcoin/pull/18772.

```suggestion
Requirements are a Bitcoin Core node v22.0 and a PostgreSQL database.
```

Reply:
Ah yes:
<img width="691" alt="Schermafbeelding 2022-09-12 om 15 03 09" src="https://user-images.githubusercontent.com/10217/189660843-16ebcd2c-b87f-4a7f-96ac-c4aa6f60c37e.png">